### PR TITLE
Update website title and homepage text

### PR DESCRIPTION
**Pull request by aktwo**
Hey man I love your website but I had some suggestions.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,8 +2,8 @@
 <html>
 
 <body>
-  <h1>Hello World</h1>
```
Comment (alstend) [inline index.html:5]: Hey why did you remove this?

Comment (aktwo) [reply index.html:5]: Oh, that's a good point. I didn't mean to do that.

Comment (aktwo) [reply index.html:5]: Will update the pull request now.

File: index.html [hunk 2]
```diff
@@ -2,8 +2,8 @@
...
-  <p>I'm doing this for fun.</p>
+  <h1>Hello world</h1>
```
Comment (alstend) [inline index.html:5]: You messed up, it should be uppercase "W".

**Review by alstend — COMMENTED** [review]
Please delete and do better next time.
